extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -110,7 +140,14 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       {name: 'u_w', value: screen ? screen.width : null},
       {name: 'vis', value: visibilityStateCodes[visibilityState] || '0'},
       {name: 'wgl', value: global['WebGLRenderingContext'] ? '1' : '0'},
-    ], []);
+    ];
+
+    if (prevFmts) {
+      paramList.push({name: 'prev_fmts', value: prevFmts});
+    }
+
+    return googleAdUrl(
```

Comment:
Test added. I took the query parameters from a 3p AdSense ad request URL and used those as my 'gold' standard. I found that A4A is missing the following: 

slotname, ea, flash, wg, bpp, bdt, fdt, idt, shb, cbv, saldr, correlator, frm, ga_vid, ga_hid, iag, icsg, dssz, mdo, mso, u_his, u_java, u_nplug, u_nmime, dff, isw, ish, ifk, oid, loc, rx, eae, pc, brdim, rsz, abl, ppjl, pfx, fuslotname, ea, flash, wg, bpp, bdt, fdt, idt, shb, cbv, saldr, correlator, frm, ga_vid, ga_hid, iag, icsg, dssz, mdo, mso, u_his, u_java, u_nplug, u_nmime, dff, isw, ish, ifk, oid, loc, rx, eae, pc, brdim, rsz, abl, ppjl, pfx, fu

And contains the following extraneous parameters:

adtest=false&c=3277130952209433&wgl=1&d_imp=1&bih=150&u_hist=1&top=http%3A%2F%2Flocalhost%3A9876&ref=http%3A%2F%2Flocalhost%3A9876%2Fcontext.html.

For now, I've skipped this test since the parameters are clearly broken.